docs/web3.eth.account.rst

```diff
@@ -83,15 +83,13 @@ account you can just generate a random number that acts as a private key.
 
 To create a private key using web3.py and command line you can do:
 
-```shell
-python -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.toHex(acc.privateKey)}, account={acc.address}')"
-```
+.. code-block:: shell
+
+    python -c "from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.to_hex(acc.key)}, account={acc.address}')"
```

Comment:
@pacrob @fselmo before I merge, wanted to double check: is `acc.privateKey` just `acc.key` now?

Reply:
```
>>> from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.to_hex(acc._private_key)}, account={acc.address}')
private key=0x9bb184a212058662a6f8384bc91cxxxxxxxxxxxxxxxxxxxxxxx account=0xde94644Adb231f69E2aF01f7FF5Ab6182Fd246cc
>>> from web3 import Web3; w3 = Web3(); acc = w3.eth.account.create(); print(f'private key={w3.to_hex(acc.key)}, account={acc.address}')
private key=0xd198207b8c323315c5d3dcc8633xxxxxxxxxxxxxxxxxxxxxxxx, account=0x90bf5A2FCDC10A1168C7DFfC2C3A69a6A3C6BC9F
```

both works I guess